contrib/completions.fish

```diff
@@ -18,7 +18,7 @@ complete -c exa        -l 'colour-scale' -d "Highlight levels of file sizes dist
 # Filtering and sorting options
 complete -c exa -l 'group-directories-first' -d "Sort directories before other files"
 complete -c exa -l 'git-ignore'           -d "Ignore files mentioned in '.gitignore'"
-complete -c exa -s 'a' -l 'all'       -d "Show and 'dot' files"
+complete -c exa -s 'a' -l 'all'       -d "Show hiden and 'dot' files"
```

Comment:
`hidden` :)

Reply:
I did this PR too fast. :sob: Thanks!

Notice there is a «suggestion» feature in GitHub now (first icon in formatting toolbar) that is very useful when reviewing code. :slightly_smiling_face: